src/Uno.Sdk.Updater.targets

```diff
@@ -1,11 +1,11 @@
 ﻿<Project>
   <!-- This file is generated by the Uno.Sdk.Updater. Do not make manual changes. -->
   <PropertyGroup>
-    <UnoVersion>5.4.0-dev.207</UnoVersion>
+    <UnoVersion>5.3.20</UnoVersion>
     <UnoWasmBootstrapVersionNet8>8.0.17</UnoWasmBootstrapVersionNet8>
     <UnoWasmBootstrapVersionNet9>9.0.0-dev.88</UnoWasmBootstrapVersionNet9>
     <UnoExtensionsLoggingVersion>1.7.0</UnoExtensionsLoggingVersion>
-    <UnoCoreLoggingVersion>4.0.1</UnoCoreLoggingVersion>
-    <UnoDspTasksVersion>1.4.0-dev.12</UnoDspTasksVersion>
+    <UnoCoreLoggingVersion>4.1.1</UnoCoreLoggingVersion>
```

Comment:
@jeromelaban This is a regression. This version is NOT available on NuGet.org. Are these automated PRs getting versions from private feed?

Reply:
Yes, this is tracked in https://github.com/unoplatform/uno.templates/issues/764